src/main/ruby/core/rbconfig.rb

```diff
@@ -37,6 +37,11 @@ module RbConfig
   host_os  = Truffle::System.host_os
   host_cpu = Truffle::System.host_cpu
 
+  # host_os for linux systems is reported as linux-gnu rather than simply linux.
+  host_os = 'linux-gnu' if host_os == 'linux'
+
+  host_vendor = 'unknown'
```

Comment:
I would move these right next to the 2 host_ variables above since they are related.